### PR TITLE
Config Fine Tune

### PR DESCRIPTION
**Pull request by stevenbennitt**

### CODE REVIEW

File: configs/bigcommerce.json [hunk 1]
```diff
@@ -1,37 +1,625 @@
 {
-  "index_name": "bigcommerce",
+  "index_name": "TESTING_INDEX",
```
Comment (ElPicador) [inline configs/bigcommerce.json]: Could you put the previous `index_name`?

File: configs/bigcommerce.json [hunk 2]
```diff
@@ -1,37 +1,625 @@
...
   "start_urls": [
-    "https://support.bigcommerce.com/",
-    "https://support.bigcommerce.com/guides",
-    "https://support.bigcommerce.com/university",
-    "https://support.bigcommerce.com/documentation"
-  ],
+	{"url":"https://support.bigcommerce.com/articles/Learning/What-is-SEO","selectors_key": "guides"},
```
Comment (ElPicador) [inline configs/bigcommerce.json]: Seems big and complicated, can't you do something simpler?
It seems each `selectors_key` is associated with a specific URL pattern.

File: configs/bigcommerce.json [hunk 3]
```diff
@@ -1,37 +1,625 @@
...
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-my-customers-reset-their-password-for-their-account-on-my-store","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-change-the-Unable-to-Ship-to-this-Location-message-on-checkout","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-auto-return-my-customers-to-my-store-after-they-checkout-with-PayPal","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-control-the-display-order-of-the-products-on-my-category-and-brand-pages","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-Bigcommerce-Calculates-Trailing-12-Month-Sales-Orders-to-Determine-Your-Pricing","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-access-and-install-the-new-single-click-apps-within-my-Bigcommerce-store-control-panel","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public
```
Comment (ElPicador) [inline configs/bigcommerce.json]: could you indent it correctly?

Comment (ElPicador) [inline configs/bigcommerce.json]: it's `js_wait`

File: configs/bigcommerce.json (hunk 2 above)
Comment (stevenbennitt) [inline configs/bigcommerce.json]: I agree and I would prefer not to do it this way but based on the way the site is set up that craw config is not getting every URL in the site.

File: configs/bigcommerce.json (hunk 3 above)
Comment (stevenbennitt) [inline configs/bigcommerce.json]: How critical is that? It would take a while to do is why I'm asking.

Comment (stevenbennitt) [inline configs/bigcommerce.json]: k, since i've had it wrong Im going to remove it.

Comment (ElPicador) [inline configs/bigcommerce.json]: can't you use a linter? like `jq`? 
It allows to have better diff when modifying the file

File: configs/bigcommerce.json (hunk 2 above)
Comment (ElPicador) [inline configs/bigcommerce.json]: Have you tried it locally? Because I'm not sure that many start_urls will work

Comment (stevenbennitt) [inline configs/bigcommerce.json]: I have, it should find the URLs on my local copy it crawls the urls without issue.

**Comment by ElPicador** [comment]
I'm sorry but it's still not indented, just use a linter even an online one like http://jsonlint.com/

**Comment by maxiloc** [comment]
Let's close this. Could you send us via email urls that are not crawled and we'll investigate

**Pull request closed without merging**